www/class/centreonConfigCentreonBroker.php

```diff
@@ -718,18 +707,14 @@ public function updateConfig($id, $values)
      */
     public function updateCentreonBrokerInfos($id, $values)
     {
-        /*
-        * Clean the informations for this id
-        */
-        $query = "DELETE FROM cfg_centreonbroker_info WHERE config_id = " . $id;
+        // Clean the informations for this id
+        $query = "DELETE FROM WHERE config_id = " . $id;
```

Comment:
```suggestion
        $query = "DELETE FROM cfg_centreonbroker_info WHERE config_id = " . $id;
```

Reply:
please bound $id and add a PDO transaction / commit / rollback
as you'll insert multiple rows in the foreach occurences... yeahYeah, it's out of your scope